src/main.js

```diff
@@ -8,6 +8,15 @@ import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap-vue/dist/bootstrap-vue.css';
 Vue.use(BootstrapVue);
 
+// Load and setup vue-cookies
+import VueCookies from 'vue-cookies'
+Vue.use(VueCookies)
+// set default config
+VueCookies.config('aw-webui')
+if (!VueCookies.isKey('start-of-day')) {
+    VueCookies.set('start-of-day','02:00');
```

Comment:
I'd suggest a default of 04:00, that's what the Oura app uses (and some other apps I've used).

Reply:
Fixed